extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -36,7 +42,7 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
     // 'limited-content' is for the paywall dialog page, where a paywall would trigger based on both time advance or click events.
     // 'content' is for all the remaining locked pages.
     iterateCursor(
-      document.querySelectorAll('amp-story-page'),
+      this.element.parentElement.getElementsByTagName('amp-story-page'),
```

Comment:
Are these necessary since we are blocking the page navigation anyway? We should maybe also thing about just using index directly?

Reply:
to clarify these attrs are not the same as what is being checked in `AmpStoryPage.isPaywallProtected()`